app/menus/CommentMenu.tsx

```diff
@@ -57,6 +59,26 @@ function CommentMenu({
     }
   }, [t, document, comment]);
 
+  const fetchReactionData = React.useCallback(async () => {
```

Comment:
This seems unnecessary, we can do any such operations as part of the render loop in the component. We're also transforming this data twice – once here and then again inside the `ViewReactions` dialog.

Lets just call `comment.fetchReactions` inside the dialog

Reply:
(there is another suggestion related to moving the transform to a computed getter further down)